README.md

```diff
@@ -1,66 +1,109 @@
--------
-### ⚠️  PLEASE READ THE [INSTRUCTIONS](/INSTRUCTIONS.md) FOR GUIDELINES ON HOW TO START YOUR PACKAGE.
-> Don't forget to remove this warning while updating this README.
--------
-
-# {package-name}
+# next-button
 
 [![NPM version][npm-image]][npm-url] [![Downloads][downloads-image]][npm-url] [![Build Status][build-status-image]][build-status-url] [![Coverage Status][codecov-image]][codecov-url] [![Dependency status][david-dm-image]][david-dm-url] [![Dev Dependency status][david-dm-dev-image]][david-dm-dev-url]
 
-[npm-url]:https://npmjs.org/package/@moxy/{package-name}
-[downloads-image]:https://img.shields.io/npm/dm/@moxy/{package-name}.svg
-[npm-image]:https://img.shields.io/npm/v/@moxy/{package-name}.svg
-[build-status-url]:https://github.com/moxystudio/{package-name}/actions
-[build-status-image]:https://img.shields.io/github/workflow/status/moxystudio/{package-name}/Node%20CI/master
-[codecov-url]:https://codecov.io/gh/moxystudio/{package-name}
-[codecov-image]:https://img.shields.io/codecov/c/github/moxystudio/{package-name}/master.svg
-[david-dm-url]:https://david-dm.org/moxystudio/{package-name}
-[david-dm-image]:https://img.shields.io/david/moxystudio/{package-name}.svg
-[david-dm-dev-url]:https://david-dm.org/moxystudio/{package-name}?type=dev
-[david-dm-dev-image]:https://img.shields.io/david/dev/moxystudio/{package-name}.svg
+[npm-url]:https://npmjs.org/package/@moxy/next-button
+[downloads-image]:https://img.shields.io/npm/dm/@moxy/next-button.svg
+[npm-image]:https://img.shields.io/npm/v/@moxy/next-button.svg
+[build-status-url]:https://github.com/moxystudio/next-button/actions
+[build-status-image]:https://img.shields.io/github/workflow/status/moxystudio/next-button/Node%20CI/master
+[codecov-url]:https://codecov.io/gh/moxystudio/next-button
+[codecov-image]:https://img.shields.io/codecov/c/github/moxystudio/next-button/master.svg
+[david-dm-url]:https://david-dm.org/moxystudio/next-button
+[david-dm-image]:https://img.shields.io/david/moxystudio/next-button.svg
+[david-dm-dev-url]:https://david-dm.org/moxystudio/next-button?type=dev
+[david-dm-dev-image]:https://img.shields.io/david/dev/moxystudio/next-button.svg
 
-{package-drescription}
+A react button that doubles down as a link.
 
 ## Installation
 
 ```sh
-$ npm install @moxy/{package-name}
+$ npm install @moxy/next-button
 ```
 
 This library is written in modern JavaScript and is published in both CommonJS and ES module transpiled variants. If you target older browsers please make sure to transpile accordingly.
 
 ## Motivation
 
-{package-motivation}
+Every website or app needs a button, so this is our bare-bones version of it. In addition, it doubles down as a [NextLink](https://github.com/moxystudio/next-link) if an `href` prop is passed, maintaining the style of a button.
```

Comment:
I would replace "it doubles down as a NextLink" with "can behave as a NextLink component"

Reply:
Same as before.